README.md

```diff
@@ -874,15 +874,7 @@ easier understanding and reading of a test.
     end
     ```
 
-* Add an example ensuring that the FactoryGirl.created model is valid.
```

Comment:
I don't think `article` here is meant to be a mock but an actual `Article` instance. We should probably remove the FactoryGirl mention, since whether it was built using a factory or not isn't really relevant to the test and it seems like it's misleading.